src/components/ebay-notice/template.marko

```diff
@@ -2,11 +2,15 @@
     w-bind
     aria-labelledby="${data.status}-status-${widget.elId()}"
     class=data.mainClass
-    ${data.htmlAttributes}>
+    ${data.htmlAttributes}
+    if(!data.hidden)>
         <${data.headingTag}
             class=data.headingClass
             id="${data.status}-status-${widget.elId()}">
                 <span aria-label="${data.ariaText}" role="img"></span>
         </${data.headingTag}>
         <${data.contentTag} class=data.contentClass w-body/>
+         <button aria-label="${data.ariaClose}" class=data.dismissedClass w-onclick="onDismiss" if(data.dismissible)>
```

Comment:
You should test the presence of this button for both dismissible and non-dismissible in the server side tests.

Reply:
okay makes sense I will add it